microk8s-resources/wrappers/microk8s.wrapper

```diff
@@ -27,6 +27,9 @@ elif [ "${APP}" == "inspect" ]; then
 elif [ "${APP}" == "help" ] || [ "${APP}" == "--help" ] || [ "$APP" == "-h" ]; then
     help
     readonly EXIT="0"
+elif [ "${APP}" == "--version" ] || [ "${APP}" == "-v" ]; then
+    "${SNAP}/microk8s-version"
```

Comment:
If we changed the name of the script to `microk8s-version.wrapper`, we would basically also get the `microk8s version` flavor of the command for free.